test/ims.test.js

```diff
@@ -136,4 +136,39 @@ describe('Verify communication with IMS', () => {
     await expect(generateAccessToken('', process.env.IMS_CLIENT_ID, 'client-secret', 'adobeid'))
       .resolves.toBe('cached-access--token');
   });
+
+  test('Verify checking that provided IMS access token is a service token with no required scopes', () => {
```

Comment:
is there a test that checks whether a `system` scope is a valid service token?

Reply:
Good catch, just added one